docs/pages/management/dynamic-resources/teleport-operator-standalone.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Standalone Kubernetes Operator
+description: Run a standalone operator against a remote Teleport cluster such as Teleport Cloud.
+---
+
+This guide explains how to run the Teleport Kubernetes Operator against any remote Teleport cluster.
+If your Teleport cluster is deployed using the `teleport-cluster` Helm chart, you might want to follow
+[the guide for Helm-deployed clusters](./teleport-operator-helm.mdx) instead.
+
+## Prerequisites
+
+(!docs/pages/includes/self-hosted-prereqs-tabs.mdx!)
+
+- a Kubernetes cluster. You must be able to create/read Namespace, ServiceAccount,
+  Deployment, Secret, Role, RoleBinding and CustomResourceDefinition resources.
+- [Helm](https://helm.sh/docs/intro/quickstart/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+Validate Kubernetes connectivity by running the following command:
+
+```code
+$ kubectl cluster-info
+# Kubernetes control plane is running at https://127.0.0.1:6443
+# CoreDNS is running at https://127.0.0.1:6443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+# Metrics-server is running at https://127.0.0.1:6443/api/v1/namespaces/kube-system/services/https:metrics-server:https/proxy
+```
+
+<Admonition type="tip">
+  Users wanting to experiment locally with the operator can use [minikube](https://minikube.sigs.k8s.io/docs/start/)
+  to start a local Kubernetes cluster:
+  ```code
+  $ minikube start
+  ```
+</Admonition>
+
+### Step 1/4 - Create the operator role
+
+In this step we create the role the operator uses to interact with Teleport resources.
+
+Download and apply the operator role manifest:
+
+```bash
+$ curl -L -O https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/integrations/operator/hack/fixture-operator-role.yaml
```

Comment:
Can we use the direct import and show the fixture instead of using curl?  
This will prevent any docs failure if someone deletes the file by mistake

Reply:
The file is quite large. It cannot really be appropriately displayed in a single page :/

Reply:
that's sad :cry: